projects/plugins/boost/compatibility/page-optimize.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+function jetpack_boost_page_optimize_js_excludes( $default ) {
+	$exclude_list = get_option( 'page_optimize-js-exclude' );
+	if ( ! $exclude_list ) {
+		return $default;
+	}
+
+	delete_option( 'page_optimize-js-exclude' );
+	add_option( 'jetpack_boost_ds_minify_js_excludes', $exclude_list );
+
+	return $exclude_list;
+}
+
+function jetpack_boost_page_optimize_css_excludes( $default ) {
+	$exclude_list = get_option( 'page_optimize-css-exclude' );
+	if ( ! $exclude_list ) {
+		return $default;
+	}
+
+	delete_option( 'page_optimize-css-exclude' );
```

Comment:
I think we can migrate data without being destructive to external plugins.

Reply:
Updated with 5db69a04a3f38081d7b8c9860b5876ee63eb5c13.